README.md

```diff
@@ -114,21 +67,87 @@ where `bin/`, `lib/` etc. are. This should be the only argument that is
 mandatory, everything else should be optional. The default build type is
 Release, though this can be overridden.
 
-The following example shows how to specify the ComputeCpp location, then
-build and run the tests.
+The following command shows how to compile SYCL-DNN.
 
 ```bash
 # Setup build environment
 mkdir build && cd build
 cmake .. -DComputeCpp_DIR=/path/to/computecpp
 # Compile SYCL-DNN
 make -j`nproc`
+```
+
+### Sample Code
+
+The "samples" directory contains sample code for the 2D convolution and pooling 
+operations offered by SYCL-DNN. 
+
+These binaries are compiled when building SYCL-DNN using CMake.
+
+### Running the SYCL-DNN Tests
+
+The SYCL-DNN tests are compiled when building SYCL-DNN using CMake.
+The following command shows how to run the tests.
+
+```bash
 # Run the tests
 ctest
 # If compiled with benchmark support, run just the benchmarks
 ctest -C Benchmark -E test
 ```
 
+## Support
+
+### Bug reports and Issues
+
+Bug reports are vital to provide feedback to the developers about what is going
+wrong with the project, you can raise these using the ["Issues"](https://github.com/codeplaysoftware/SYCL-DNN/issues) 
+feature in GitHub.
+
+Please make sure that your bug report contains the following information:
+
+* A clear and descriptive title.
+
+* The output of
+  `clinfo | grep -E "Platform ID|Name|Vendor|[Vv]ersion|Profile|Extensions"`.
+
+* The output of `computecpp_info`.
+
+* The exact steps and commands to run to reproduce the bug.
+
+* The exact error text shown (if applicable), otherwise the behaviour you
+  expected and what you encountered instead.
+
+* Should the problem arise outside the project's test suite then please provide
+  a minimal test to allow us to reproduce the problem.
+
+## Project layout
```

Comment:
I'm not 100% sure this really provides much value as part of the readme. @DuncanMcBain any opinions or could this be removed/pushed to a separate `docs/` file?

Reply:
Tend to agree it has limited usefulness.